spec/rails_helper.rb

```diff
@@ -101,6 +101,9 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  config.include FactoryGirl::Syntax::Methods
```

Comment:
I try to shy away from this, particularly with a codebase with newer developers. It lets you call `create` instead of `FactoryGirl.create`. A nice shorthand, unless you don't know what `let(:o) { create :organization }` is doing